web/client/components/map/leaflet/DrawSupport.jsx

```diff
@@ -125,40 +189,43 @@ class DrawSupport extends React.Component {
             }
         });
         this.props.map.addLayer(vector);
-        // Immidiatly draw passed features
+        // Immediatly draw passed features
```

Comment:
what? isn't not correct? https://www.google.it/search?q=Immediately&oq=Immediately&aqs=chrome..69i57j0l5.1719j0j7&sourceid=chrome&ie=UTF-8